File: scratch/get_ata_fandom.js
const https = require('https');
const options = {
  hostname: 'honorofkings.fandom.com',
  path: '/api.php?action=query&titles=Ata&prop=images&format=json',
  headers: {
    'User-Agent': 'Mozilla/5.0'
  }
};
https.get(options, (res) => {
  let body = '';
  res.on('data', chunk => body += chunk);
  res.on('end', () => {
    console.log(body);
  });
});

File: scratch/get_ata_fandom.py
import urllib.request
import re

req = urllib.request.Request(
    'https://honorofkings.fandom.com/wiki/Ata',
    headers={'User-Agent': 'Mozilla/5.0'}
)
try:
    with urllib.request.urlopen(req) as response:
        html = response.read().decode('utf-8')
        matches = re.findall(r'src="([^"]*(?:ata|Zhu_Bajie)[^"]*\.png)"', html, re.IGNORECASE)
        print('Ata images:', [m for m in matches])
except Exception as e:
    print('Error:', e)
